components/runtime.rst

```diff
@@ -2,7 +2,7 @@ The Runtime Component
 =====================
 
     The Runtime Component decouples the bootstrapping logic from any global state
-    to make sure the application can run with runtimes like PHP-PM, ReactPHP,
+    to make sure the application can run with runtimes like PHP-FPM, ReactPHP,
```

Comment:
I think this change is wrong and PHP-PM refers to the [PHP Process Manager](https://github.com/php-pm/php-pm).

Reply:
@Nyholm Can you help here as you contributed the initial version of this document?

Reply:
Correct, It is meant to say PHP-PM.

Reply:
Sorry about this! I just merged your PR fixing this.